### ISSUE TITLE
Getting errors when having a helper with the same name as a component

### ISSUE THREAD

**Issue by emilkarl** [lifeart/ember-ast-hot-load#740]
I have a component `<Icon>` and there is also a helper inside [ember-leaflet ](https://github.com/miguelcobain/ember-leaflet) called `icon`.

Now Im getting weird behaviours because of this. For example I can not to `<span ...attributes>` inside my Icon component because `ast-hot-reload` is treating it as a helper.

Any ideas how to fix it, without renaming my component? I don’t need hot reloading for the `icon` helper as it will "never" change :D 

If I turn off `ember-ast-hot-reload` everything works as expected.

This is a warning I get in the console

```
Oops, looks like helper "Icon" invoked like component (due to 'ember-ast-hot-load' ast transformation).
Don't worry it's expected behavour because helper "Icon" looks like component ( {{Icon}} or <Icon />)
to fix this issue, add "Icon" into "ember-cli-build.js" in application config section

/////////////////////////////////////
let app = new EmberApp(defaults, {
  'ember-ast-hot-load': {
      helpers: ["Icon"],
      enabled: true
  }
});
/////////////////////////////////////
```

**Comment by lifeart** [comment]
Hi @emilkarl! Try to replace `helpers: ["Icon"]` with `helpers: ["icon"]` in `ember-ast-hot-load` config

**Comment by emilkarl** [comment]
I already have `icon` in there, it doesn't help. I guess the easiest thing would be to rename my component but I would like to avoid that :D

**Comment by lifeart** [comment]
@emilkarl do you use latest ast-hot-load version? error may be related to https://github.com/lifeart/ember-ast-hot-load/pull/717

If you have only console warnings, it's fine

**Comment by emilkarl** [comment]
Yes, we’re on 3.0.0. Is there some way to ignore an helper? The icon helpers comes from a different package so I could just ignore it in the hot reload. Tried looking through the code but couldnt find a way to do it. I always want `Icon` to be treated as a component.
